### PR TITLE
User story1

### PR DESCRIPTION
**Pull request by NimeshR**

### CODE REVIEW

File: src/test/java/com/google/nimesh/GrettingControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import org.springframework.boot.test.context.SpringBootTest;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@SpringBootTest
+public class GrettingControllerTest {
+
+    @Test
+    public void testController(){
+        assertTrue(true);
```
Comment (sanjogramanarayanan) [inline src/test/java/com/google/nimesh/GrettingControllerTest.java:13]: Using assertTrue(true) is not a good test

**Review by sanjogramanarayanan — COMMENTED** [review]
Fix the tests

File: src/test/java/com/google/nimesh/GrettingControllerTest.java (hunk 1 above)
Comment (NimeshR) [reply src/test/java/com/google/nimesh/GrettingControllerTest.java:13]: I want to use this. Please approve

**Comment by sanjogramanarayanan** [comment]
Looks good

**Pull request merged**